app/lib/services/secrets_service.rb

```diff
@@ -1,10 +1,22 @@
 module Services
-  class CertificateService
+  class SecretsService
     class << self
       def issue_cert(cert_issue_request)
         impl.issue_cert(cert_issue_request)
       end
 
+      def kv_read(path)
```

Comment:
SRP calling, do we want to conflate certs and kvs? Services::KeyValue.read(path) looks more railsish, no?

Reply:
The idea was this is a facade to the backend "secrets" provider (Vault or KMS) -- so if we have multiple Services classes, each will need to swap in the provider. Would the VaultService also need to break apart?

Reply:
Well. maybe but it doesn't seem like it would be an intensive refactor, so we can defer that until later.